### DISCUSSION TITLE
Help with docker compose and nginx reverse proxy on another domain

### DISCUSSION THREAD

**Discussion by astrakid** [OpenSignLabs/OpenSign#1342] [Q&A]
hi,
i have opensign up and running on my local server.
now i want to have anotther reverse proxy (nginx) in font of it. but i am neither able to do it directly fom nginx (passing caddy) nor coming over by nginx over caddy to opensign completely.
i am able to login, but as soon as i do something with a document (sign or create a template) i get an error message "something went wrong, please try again later".

my domain: is lets say privatedomain.com, and i want to use subdomain sign for this.
my local server-fqdn is test.local

PUBLIC_URL=https://sign.privatedomain.com/
GENERATE_SOURCEMAP=false
REACT_APP_SERVERURL=http://sign.test.local:3000/app
REACT_APP_APPID=opensign
REACT_APP_GTM=GTM-123N7LD

HOST_URL=http://sign.test.local

APP_ID=opensign
appName=open_sign_server
MASTER_KEY=xyz
MONGODB_URI=mongodb://mongo-container:27017/OpenSignDB
PARSE_MOUNT=/app
SERVER_URL=http://sign.test.local:3000/app

that was working with caddy locally, but not with my public fqdn, so i wanted to expose that service for some colleagues temporarely ti the internet. but with this configuration it is not working.

any ideas?

kind regards,
andre

**Comment by astrakid** [comment]
i guess it is because the documents seem to be referenced by https://sign.test.local/app/files instead of https://sign.privatedomain.com/...
any idea what i have to configure in nginx to get that corrected?

good request in chrome logs (before the error appears):
![image](https://github.com/user-attachments/assets/fd631d3b-9551-4e52-aca5-d030733be9b9)

bad request:
![image](https://github.com/user-attachments/assets/f534c887-2d69-4a47-999d-5f5c01989d63)

**Comment by moblesmarti** [comment]
Were you able to resolve this?
thanks,

**Comment by astrakid** [comment]
no. and no reaction from anyone. i had to switch to another open source tool.

**Comment by nxglabs** [comment]
@moblesmarti We’d love for you to join our [Discord server](https://discord.com/invite/xe9TDuyAyj) to get support for this and similar issues.

@astrakid We're sorry to hear about the trouble with your OpenSign installation. At the moment, we officially support only Caddy as a proxy. However, we plan to include official support for Nginx in future updates. That said, several members of our Discord community have successfully configured OpenSign with Nginx, and many are eager to help others with similar setups.
